daprdocs/content/en/getting-started/install-dapr-kubernetes.md

```diff
@@ -107,31 +107,34 @@ The latest Dapr helm chart no longer supports Helm v2. Please migrate from helm
 ### Add and install Dapr helm chart
 
 1. Make sure [Helm 3](https://github.com/helm/helm/releases) is installed on your machine
+
 2. Add Helm repo and update
 
     ```bash
     helm repo add dapr https://dapr.github.io/helm-charts/
     helm repo update
+    # See which chart versions are available
+    helm search repo dapr --devel --versions
     ```
 
-3. Create `dapr-system` namespace on your kubernetes cluster
+3. Install the Dapr chart on your cluster in the `dapr-system` namespace.
 
     ```bash
-    kubectl create namespace dapr-system
+    helm upgrade --install dapr dapr/dapr \
+    --version=1.0.0 \
```

Comment:
1.0.0 is not out. yet, we should probably change this to 1.0.0-rc.3

Reply:
thanks! fixed.